app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardFragment.kt

```diff
@@ -240,10 +240,10 @@ class DashboardFragment : BaseFragment(R.layout.fragment_dashboard) {
                 controller.uploadMissingExposureKeys = it
             }
 
-        disposables += viewModel.observeCurrentChangelogSeen()
+        disposables += viewModel.observeCurrentChangelogState()
             .observeOnMainThread()
-            .subscribe { seen ->
-                if (seen) {
+            .subscribe { changelogState ->
+                if (changelogState is ChangelogState.CurrentChangelogSeen) {
```

Comment:
Does this fail if a fresh install of a version without associated changelog is made?
I guess it's fine to check for  != `Pending` as we have the check for `wasExposureFrameworkAutomaticallyEnabledOnFirstStart.not()` below

Reply:
No, for this case we have auto start in init method of `DashboardViewModel`.